internal/cli/atlas/livemigrations/create.go

```diff
@@ -36,8 +36,8 @@ func (opts *CreateOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var createTemplate = `ID	PROJECT ID	SOURCE PROJECT ID	STATUS
-{{.Id}}	{{.Destination.GroupId}}	{{.Source.GroupId}}	{{.Status}}`
+var createTemplate = `ID	LAGTIME	READY FOR CUTOVER	STATUS
+{{.Id}}	{{.LagTimeSeconds}}	{{.ReadyForCutover}}	{{.Status}}`
```

Comment:
Updated the template as the API does not send Source and Destination in the response object.